packages/web/hooks/use-swap.ts

```diff
@@ -518,16 +509,16 @@ function useSwapAsset<TAsset extends Asset>(
   );
   const queryEnabled =
     Boolean(minDenomOrSymbol) && !isLoadingWallet && !existingAsset;
```

Comment:
I believe using a Boolean for this purpose might lead to an unusual edge case. For instance, if a user searches for the string "0", the result would incorrectly be interpreted as false

```suggestion
    !isNil(minDenomOrSymbol) && typeof minDenomOrSymbol === "string" && !isLoadingWallet && !existingAsset;
```